src/Layout/LayoutColumn/index.js

```diff
@@ -1,14 +1,27 @@
 // @flow
 import * as React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 import defaultTheme from "../../defaultTheme";
 import getViewportHideStyles from "../../Hide/helpers/getViewportHideStyles";
+import { StyledCard } from "../../Card";
+import mq from "../../utils/mediaQuery";
 
 import type { Props } from "./index";
 
 const StyledColumn = styled.div`
   ${({ hideOn }) => !!hideOn && getViewportHideStyles(hideOn)};
+
+  ${StyledCard} {
+    margin: 0 -${({ theme }) => theme.orbit.spaceMedium};
+    width: auto;
+  }
+  ${mq.tablet(css`
+    ${StyledCard} {
+      margin: auto;
```

Comment:
Is there a reason for setting to `auto` instead of `0`? `0` is the initial value of `margin`.

```suggestion
      margin-left: 0;
      margin-right: 0;
```

Reply:
As I'm thinking about it, we should not use either `auto` or `0`, as it can be problematic for some cases. Instead of that, we should implement the other way of media query – `max-width` so we don't have to set up the default value once again, we don't even know what the value will be.

Reply:
I like this solution as well ✅